pkg/log/auditlog/types.go

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (C) 2024 IOTech Ltd
+//
+
+package auditlog
+
+import "time"
+
+// ActionType is a categorical identifier used to give high-level insight as to the action type.
+type ActionType string
+
+// Constant ActionType identifiers which can be used to label and group audit log by their action type.
+const (
+	ActionTypeAdd      ActionType = "Add"
+	ActionTypeCreate   ActionType = "Create"
```

Comment:
what the difference between Add and Create?

Reply:
Maybe we can keep `Create` and add `Add` back if needed.